skyvern/webeye/actions/handler.py

```
@@ -3451,6 +3451,20 @@ async def normal_select(
     index: int | None = json_response.get("index")
     value: str | None = json_response.get("value")
 
+    try:
+        await locator.click(
+            timeout=settings.BROWSER_ACTION_TIMEOUT_MS,
+        )
+    except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)